src/main/java/com/openlattice/controllers/OrganizationsController.java

```diff
@@ -125,10 +95,9 @@
                         .map( AuthorizationUtilsKt::getLastAclKeySafely )
         );
 
-        return StreamSupport.stream( orgs.spliterator(), false ).map( org -> {
-            org.getRoles().removeIf( role -> !authorizedRoles.contains( role.getAclKey() ) );
-            return org;
-        } ).collect( Collectors.toList() );
+        return StreamSupport.stream( orgs.spliterator(), false ).peek( org ->
```

Comment:
`map()` should work just fine, so why use `peek()`? The docs say:

> This method exists mainly to support debugging, where you want to see the elements as they flow past a certain point in a pipeline:

https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/stream/Stream.html#peek(java.util.function.Consumer)

Reply:
to fix the compiler warnings (since we're not returning a different object that's being mapped, we're only modifying it, the compiler gets angry)